.github/workflows/ci_doctests.yaml

```diff
@@ -0,0 +1,106 @@
+# This workflow installs PyGMT and runs all doctests
+
+name: Doctests
+
+on:
+  # push:
+  #   branches: [ main ]
+  pull_request:
+  # Schedule weekly tests on Sunday
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  test:
+    name: ${{ matrix.os }} - Python ${{ matrix.python-version }} / NumPy ${{ matrix.numpy-version }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.9', '3.11']
+        os: [ubuntu-latest, macOS-latest, windows-latest]
+        # Pair Python 3.9 with NumPy 1.21 and Python 3.11 with NumPy 1.24
+        include:
+          - python-version: '3.9'
+            numpy-version: '1.21'
+          - python-version: '3.11'
+            numpy-version: '1.24'
```

Comment:
Do we need to do tests on both Python 3.9 and 3.11? Or can we just do one test (e.g. on Python 3.11)?

Reply:
I'm OK with one Python version.